extension-workspace-reference.md

```diff
@@ -0,0 +1,121 @@
+# workspace/xreference extension to LSP
```

Comment:
The request should be plural (`workspace/xreferences`) to be consistent with `textDocument/references`